### ISSUE TITLE
Shorten the schedule day

### ISSUE THREAD

**Issue by MikeRawlins** [netmix/radio-station#420]
Is possible or can it be a feature to remove/shorten the schedule?

I am working with a radio station that has live presenters 0700 - 2200 and then a music mix, I'd like to remove hours 0000 - 0700 and 2200 - 0000 each day

**Comment by kairandles** [comment]
You should be able to do 22:00 to 07:00 I have shows that run from 10:00 till 02:00

**Comment by MikeRawlins** [comment]
> You should be able to do 22:00 to 07:00 I have shows that run from 10:00 till 02:00

Yes I can do that, but I actually want to hide those hours to make the schedule table/list shorter. I'd be happy with the schedule just showing the hours 0700 - 2200 and I could use a show widget to show 'Music mix'

**Comment by kairandles** [comment]
Oh I mean you don't have to schedule for all hours? Widgets will just say that no show is scheduled. I would say though that what you are saying is a bit counter intuitive. The schedule shows the listener what shows are scheduled so leaving shows off this list seems a bit strange to me if you want the widgets showing them?!

**Comment by majick777** [comment]
@MikeRawlins Sorry for the delay I seem to have missed this issue somehow.
The easiest way to do this would be to use CSS to hide those hour rows on the master schedule table: 

`.master-program-hour-row.hour-row-0, .master-program-hour-row.hour-row-1, .master-program-hour-row.hour-row-2, .master-program-hour-row.hour-row-3, .master-program-hour-row.hour-row-4, .master-program-hour-row.hour-row-5, .master-program-hour-row.hour-row-6, .master-program-hour-row.hour-row-22, .master-program-hour-row.hour-row-23 {display: none;}`

We had considered implementing something in the shortcode itself in the past to not process those hours, but it is not technically a good idea since a show can start any time and be for any length... so just hiding the display of the rows is really a better idea for this... Possibly we could add an attribute to the shortcode that automatically adds the CSS.
eg. `[master-schedule hide_hours="0,1,2,3,4,5,6,22,23"]` ... I will repurpose this ticket to add that.
